soroban-env-common/env.json

```diff
@@ -120,6 +120,13 @@
                     "args": [],
                     "return": "AddressObject",
                     "docs": "Get the Address object for the current contract."
+                },
+                {
+                    "export": "a",
+                    "name": "get_max_entry_expiration",
+                    "args": [],
+                    "return": "U32Val",
+                    "docs": "Return the maximum number of ledgers from the current ledger that an entry can live for."
```

Comment:
We probably should also mention that the equation is `current ledger + max ledger - 1`.

Reply:
Updated to expose `max_expiration_ledger` instead.